CloudinaryDotNet/Cloudinary.AdminApi.MetadataFields.cs

```diff
@@ -149,6 +150,35 @@ public MetadataUpdateResult UpdateMetadata(MetadataUpdateParams parameters)
             return result;
         }
 
+        /// <summary>
+        /// Sorts metadata field datasource
```

Comment:
Please fix the doc string.

The reference can be seen on:
https://github.com/cloudinary/cloudinary_php/blob/3ed292f1f352c036f5466ee9ca8ffb018fbd8de2/src/Api/Admin/MetadataFieldsTrait.php#L185